CONTRIBUTING.md

```diff
@@ -13,8 +13,9 @@
 
 1. Fork this repository
 2. Create your feature branch, if applicable name it after the specific JIRA issue being fixed.
-3. Create a pull request against `develop` from your forked branch.
-4. Ensure someone else review's the pull request and that person should be the one to merge the PR.
+3. Try to keep commits organized by `squashing` closely related commits. If it makes sense format your commit in the following way: `<type>(<JIRA>): <subject><BLANK LINE><body> or link to the JIRA` [^1] .
```

Comment:
What does the additional overhead (of defining commit type and squashing for a clean commit history) buy us?
Having the Jira issue ID makes sense to me, and I would think that the Jira issue would provide a full description of the change type.

Reply:
squashing makes the history a little cleaner/shorter and easier to read. Same with commit type. e.g. if it's just a style and you are looking for a code change you will skip it.
But I agree that sometimes there is not a clear distinction.
So then` (<JIRA>): <subject><BLANK LINE><body if needed>` ? works for me

Reply:
I think using that commit message format would be a great addition, it makes commit messages much more valuable. And it makes writing the description of the pull request easier because you can read the commit messages. I'm hesitant with commit squashing though, I can see the argument for it, but I would make the case that it should be a contributor preference.

Reply:
@leem42 re:squashing yes, I should have communicated more clearly. It's definitely  should be more of a 'be mindful of ..' situation rather than a rule